### ISSUE TITLE
Python send to REPL blank lines problem

### ISSUE THREAD

**Issue by pszynk** [hlissner/doom-emacs#4375]
This is the same issue as kaz-yos/eval-in-repl#15

**What did you expect to happen?**
```python
def test_function(t):
    print(t)

    print(t)

print("ok")

test_function("ok")
```

select code and call `+python/open-ipython-repl`:

Should evaluate without problems just like when code is evaluated **without** sending it to REPL.

**What actually happened?**
 
```
>>> def test_function(t):
...     print(t)
... 
>>>     print(t)
  File "<stdin>", line 1
    print(t)
IndentationError: unexpected indent
>>> 
>>> print("ok")
ok
>>> 
>>> test_function("ok")
ok
>>> 
>>> 
```

Don't know if there's a smart way to resolve this.
This is just how REPL behaves when a blank line is inserted.
If instead of blank line a `TAB` is inserted this evaluates as expected.

**Comment by mnicstruwig** [comment]
To add to this, it appears that this specifically happens when using the `+eval:$command` family of commands. Using the `python-shell-send-region` command *does not* suffer from this issue.

So there seems to be a discrepancy between what, for example,  `+eval:region` is doing and `python-shell-send-region`.

It would be amazing to have this fixed!

**Comment by hlissner** [comment]
Sorry for the tremendously late response. This issue should be resolved as of #5908 (thanks to @ethan-leba). Let me know if the issue persists and I'll reopen this. Thanks for bringing it to my attention!
